recipes/extract_model/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "extract_model" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/axiom-data-science/extract_model/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5301497bb31089cebe546aaee31b1c4c241f5e1fd45f66bd814f7977dc205e65
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
I think you can make this `noarch: python` and remove this line:
```suggestion
```
It's fine if Windows then fails with missing depenedencies

Reply:
Oh ok! I thought I had to skip it because of the failure. I'll update that, thanks.